packages/calcite-components/src/tests/commonTests.ts

```diff
@@ -30,9 +30,13 @@ function isHTML(tagOrHTML: string): boolean {
 
 function getTag(tagOrHTML: string): ComponentTag {
   if (isHTML(tagOrHTML)) {
-    const regex = /[>\s]/;
+    const regex = /<calcite-[a-z0-9-]+/i;
```

Comment:
* `regex` was there already, but it might be good to rename it to express what it'll match.
* `match` could also be renamed to better convey its purpose.
* The error message could have more info on why it didn't match (e.g., no calcite-component tag used). Also, WDYT about throwing if it finds more than 1? Might not happen much, but if it does, it'll be useful to have this early warning when using the helper.